test/house_test.rb

```diff
@@ -0,0 +1,128 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/house.rb"
+require "./lib/room.rb"
+
+class HouseTest < Minitest::Test
+  def test_it_exists
```

Comment:
You should consider using a `setup` method with instance variables to create your objects for this test so that you can avoid duplicating work and DRY this up a bit. It will also save you precious time in these challenges!